serverless-api/src/repository/places.ts

```diff
@@ -41,6 +41,24 @@ export const getPlaceByName = async (
   return res.Items[0] as Place | undefined;
 };
 
+export const getPlaceByGoogleId = async (
+  googlePlaceId: string
+): Promise<Place | undefined> => {
+  const queryParams = {
+    TableName: "Places",
+    IndexName: "placeGoogleIdIndex",
+    KeyConditionExpression: "googlePlaceId = :google_place_id",
+    ExpressionAttributeValues: { ":google_place_id": googlePlaceId }
+  };
+  const res = await dynamodb.query(queryParams).promise();
+
+  if (!res.Items || !res.Items[0]) {
```

Comment:
```suggestion
  if (!res.Items) {
```

Undefined check in 2 places